source/adminguide/networking/security_groups.rst

```diff
@@ -41,8 +41,7 @@ desired set of rules.
 Any CloudStack user can set up any number of additional security groups.
 When a new instance is launched, it is assigned to the default security group
 unless another user-defined security group is specified. An instance can be a
-member of any number of security groups. Once an instance is assigned to a
-security group, it remains in that group for its entire lifetime; you
+member of any number of security groups. You change the security groups of an instance in a stopped state; you
```

Comment:
You can change the security groups of an instance only when it's in a stopped state